libmscore/chord.cpp

```diff
@@ -2285,11 +2285,9 @@ void Chord::layoutTablature()
                   }
             }
       if (!tab->genDurations()            // if tab is not set for duration symbols
-            || track2voice(track())       // or not in first voice
-            || isGrace()) {               // or chord is a grace (no dur. symbols for graces
```

Comment:
Well, you are much more familiar with lute literature than I do. I think it would be valuable to have an idea of:
1) how much of the lute literature does not use grace notes at all (I suspect the whole 'early' literature falls in this area): for them, the treatment of grace notes would be irrelevant.
2) how much uses grace notes, but do not show their duration flags
3) how much uses grace notes AND show their duration flags

I think that the relative size of 2) vs. 3) would help in deciding if it would be better to have the flags always shown by default (hiding them manually in the fewer cases where they should not appear) OR to add a new style variable to turn them off for the whole score, if 2) is "frequent enough".

The whole thing is possibly suitable to a forum topic; in the meantime, if you -- as an advanced user of these very features -- are satisfied, it is ok with me (for what this is worth!). The only caveat is to remember to keep in mind the whole field of application, rather than focusing on one's own personal and current interests (it happens to me constantly! ;) ).